tests/python/gaia-ui-tests/gaiatest/apps/marketplace/app.py

```diff
@@ -70,3 +72,57 @@ def tap_install_button(self):
 
     def get_app_name(self):
         return self.root_element.find_element(*self._name_locator).text
+
+class MarketplaceDev(Marketplace):
+
+    name = 'Dev'
+    manifest_url = 'https://marketplace.firefox.com/app/bee10fbb-7829-4d99-8a9f-a6d0a905138e/manifest.webapp'
+
+    _enable_addons_locator = (By.ID, 'enable-addons')
+
+    def _enable_addons(self):
+        element = Wait(self.marionette).until(
+            expected.element_present(*self._enable_addons_locator))
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
+
+    def _open_addon(self, addon_name):
+        selector = '//a[text()="%s"]' % addon_name
+        addon_locator = (By.XPATH, selector)
+
+        element = Wait(self.marionette, timeout=20, interval=1).until(
+            expected.element_present(*addon_locator))
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
+        return AddonPage(self.marionette)
+
+    def install_addon(self, addon_name):
+        self.search(':debug')
+
+        # Debug page is opened, tap on the "Enable Firefox OS Add-ons" button
```

Comment:
"tap on the "Enable Firefox OS Add-ons" button" is not necessary. I would move "Debug page is opened" before, so we have something like:
```python
# Open debug page
self.search(':debug')
```